test/fixtures/cookbooks/languages_node/recipes/default.rb

```diff
@@ -0,0 +1,4 @@
+include_recipe 'chef-sugar::default'
+include_recipe 'languages::default'
+
+node_install 'v4.1.2'
```

Comment:
Just install two versions of node (default and non-default), and then test for both existing in the file below.